src/reducer.js

```diff
@@ -0,0 +1,36 @@
+const initialState = {
+  newId: 100,
+  listNm: [
+    { id: 1, title: '마녀주방' },
+  ],
+  listSort: [
+    { id: 1, title: '한식' },
+  ],
+  listAddr: [
+    { id: 1, title: '서울시' },
+  ],
+};
+
+export default function reducer(state = initialState, action) {
+  if (action.type === 'updateList') {
+    return {
+      ...state,
```

Comment:
```suggestion
      state,
```
이 경우에는 그냥 state 와 같이 처리해도 될 것 같아요.